.github/ISSUE_TEMPLATE/0-bug-report.md

```diff
@@ -6,9 +6,17 @@ assignees: ''
 
 ---
 
+## Hello!
+<!-- Please leave this section as-is, it's designed to help others in the community know how to interact with our GitHub issues. -->
+
+- Please feel free to vote on this issue by adding a 👍 reaction; this helps the Pulumi team prioritize this request
+- If you're interested in contributing a fix for this issue, please leave a comment (and a link to your pull request, if you've opened one already)
```

Comment:
```suggestion
- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment
```

don't feel strongly other than if we keep it short we have a better chance that folks will read it

Reply:
```suggestion
- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)
```